api/python/src/cellxgene_ontology_guide/supported_versions.py

```diff
@@ -57,6 +57,9 @@ class CXGSchema:
     """The schema version used by the class instance."""
     supported_ontologies: Dict[str, Any]
     """A dictionary of supported ontologies for the schema version."""
+    imported_ontologies: Dict[str, str]
+    """A dictionary of ontologies with terms imported into supported ontologies, mapped to the supported ontology
```

Comment:
what if it was rephrased to "A dictionary of 'additional ontologies' mapped to the supported ontologies they are imported in"? 

the idea is if ZFS is part of ZFA, then this would return {"ZFS":"ZFA"}

Reply:
imported is the word owl uses to refer to ontologies within other ontologies

Reply:
i'd suggest explicitly including the example in the documentation. maybe something like:

For a given ontology that has been imported into another ontology, this dictionary maps from the 'additional ontology' into the supported ontology term. For example, for zebrafish, ZFS ontology terms were imported into the ZFA ontology, so `imported_ontologies` would return `{"ZFS":"ZFA"}`